PropertyInspector/PluginActionPI.html

```diff
@@ -32,9 +32,22 @@
                 <div class="sdpi-item-label">Timer Name</div>
                 <input class="sdpi-item-value sdProperty" id="timerName" value="" placeholder="Enter the name of the Timer" oninput="setSettings()">
             </div>
-            <div class="sdpi-item">
-                <div class="sdpi-item-label">Server Url</div>
-                <input class="sdpi-item-value sdProperty" id="serverUrl" value="https://api.clockify.me/api/v1" placeholder="Enter the ULR of the Server" oninput="setSettings()" required>
+            <div type="group" class="sdpi-item" id="advanced">
+                <div class="sdpi-item-label">Advanced</div>
+                <div class="sdpi-item-group" id="messagegroup_items">
+                    <div class="sdpi-item">
+                        <div class="sdpi-item-label">Title Format</div>
+                        <div class="sdpi-item-value textarea">
+                            <textarea class="sdpi-item-value sdProperty" id="titleFormat" type="textarea" maxlength="100" oninput="setSettings()">{projectName}:
+{taskName}
+{timer}</textarea>
```

Comment:
Can you indent that to line up with the rest of the HTML, the spaces are ignored, so no need to worry about that

```suggestion
                            <textarea class="sdpi-item-value sdProperty" id="titleFormat" type="textarea" maxlength="100" oninput="setSettings()">
                                {projectName}:
                                {taskName}
                                {timer}
                            </textarea>
```